packages/config-plugins/src/ios/Updates.ts

```diff
@@ -71,6 +52,7 @@ export const withUpdates: ConfigPlugin<{ expoUsername: string | null }> = (
 };
 
 export function setUpdatesConfig(
+  projectRoot: string,
```

Comment:
Make sure to mention this in the Changelog as it breaks: https://github.com/expo/eas-cli/blob/23237210d1485ce5b1afeb6146a5d0f8e2f4475a/packages/eas-cli/src/build/ios/UpdatesModule.ts#L17